### ISSUE TITLE
PATH_FILTER is deprecated

### ISSUE THREAD

**Issue by mdavidsaver** [epicsdeb/seq#1]
make 4.0 complains:

```
/etc/epics/configure/rules.d/seqsnc.make:69: PATH_FILTER is deprecated; used for ../sncBeamScheduler.stt
```

**Comment by ralphlange** [comment]
I had mentioned that to @bfrk (upstream).
Answer was: the snippet is intended to work against multiple versions of EPICS and in different contexts. If we come up with a better, working and portable solution, he is willing to take it.

**Comment by mark0n** [comment]
This has been fixed upstream with release 2.2.4 (see [release notes](https://www-csr.bessy.de/control/SoftDist/sequencer/ReleaseNotes-2-2.html#release-2-2-4)). I have packaged 2.2.6. As soon as @aderbenev has reviewed the PR we should be ready to merge.

**Comment by mark0n** [comment]
2.2.6 has been merged, closing this issue.
